Ginger/Ginger/WizardLib/WizardWindow.xaml.cs

```diff
@@ -38,11 +38,19 @@ public partial class WizardWindow : Window, IWizardWindow
 
         List<ValidationError> mValidationErrors = new List<ValidationError>();
 
-        public static void ShowWizard(WizardBase wizard, double width = 800)
+        public static void ShowWizard(WizardBase wizard, double width = 800, eWindowShowStyle windowStyle = eWindowShowStyle.Dialog)
         {
             WizardWindow wizardWindow = new WizardWindow(wizard);
             wizardWindow.Width = width;
-           wizardWindow.ShowDialog();
+            if(windowStyle == eWindowShowStyle.Dialog)
+            {
+                wizardWindow.ShowDialog();
+            }
+            else
+            {
+                wizardWindow.Topmost = true;
+                wizardWindow.Show();
+            }
```

Comment:
change the parmeter to be boolean like "DoNotShowAsDialog = false" so if-else will be based on it. this becuase you not really suppporting eWindowShowStyle options

Reply:
Changed